### ISSUE TITLE
App doesn't work at all when offline

### ISSUE THREAD

**Issue by edwardgalligan** [singuerinc/overlay#170]
It appears from [this line](https://github.com/singuerinc/overlay/blob/master/app/utils/analytics.ts#L39) that analytics is enabled by default (presumably there's some option in settings to disable it, but this can only be done *after* running the app), so if the user is online, analytics will be collected (against their will?) before they have an opportunity to turn off the setting.

The further technical issue though seems to be that if the user *isn't* online, the analytics calls fail and this results in the app not functioning.

My personal preference here would be disabling analytics by default (it's legally questionable in some regions without a ToS consent screen on first-run, but legality aside it gives a very poor first-impression). I opt-in to analytics for apps I know & love but that's never the case on first-run of a new app. Prompting users to enable analytics later could be a happy compromise.

If on-by-default analytics is a must though, I guess the app should at least work when they can't connect to a server.

**Comment by singuerinc** [comment]
@edwardgalligan I agree with you 100%. When I wrote Overlay I didn't really think about using analytics in any way. Actually... I can just remove them. The issue is that sometime ago I tried to compile the app without success and I don't have the time to figure out what's going on. I would like to re-write it TBH. But there is one issue https://github.com/electron/electron/issues/1335 created 5 years ago that still there and I really don't want to make all the effort if that issue persists.
